ICAFIX/hcp_fix_multi_run

```diff
@@ -859,13 +866,24 @@ for fmri in $fmris ; do
 	for readme_fmri in ${fmris} ; do
 		echo "  ${readme_fmri}" >> ${readme_for_cifti_out}
 	done
-		
+
 	volume_out=${fmriNoExt}_hp${hp}_clean.nii.gz
 	${Caret7_Command} -volume-merge ${volume_out} -volume ${ConcatFolder}/${concatfmrihp}_clean.nii.gz -subvolume ${Start} -up-to ${Stop}
 	fslmaths ${volume_out} -div ${ConcatFolder}/${concatfmrihp}_vn -mul ${fmriNoExt}_hp${hp}_vn -add ${fmriNoExt}_mean ${volume_out}
+
+  if [ -f ${ConcatFolder}/${concatfmrihp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean.nii.gz ] ; then
+    ${Caret7_Command} -volume-merge ${fmriNoExt}_hp${hp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean.nii.gz -volume ${ConcatFolder}/${concatfmrihp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean.nii.gz -subvolume ${Start} -up-to ${Stop}
+    fslmeants -i ${fmriNoExt}_hp${hp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean.nii.gz -m ${ConcatFolder}/${concatfmrihp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean_mask.nii.gz -o $(dirname $fmri)/Movement_Regressors_hp${hp}_clean.txt --showall
+    cat $(dirname $fmri)/Movement_Regressors_hp${hp}_clean.txt | tail -$(fslval ${fmriNoExt}_hp${hp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean.nii.gz dim4) > $(dirname $fmri)/Movement_Regressors_hp${hp}_clean.txt
+  fi
```

Comment:
Given that `${ConcatFolder}/Movement_Regressors_hp${hp}_clean.txt` has already been created, a seemingly much simpler way to accomplish this would be:
```
outFile=$(dirname $fmri)/Movement_Regressors_hp${hp}_clean.txt
sed -n "${Start},${Stop}p" ${ConcatFolder}/Movement_Regressors_hp${hp}_clean.txt > ${outFile}
```

Reply:
I wanted the intermediate NIFTI file in place with the current implementation.

Reply:
Why do we want a NIFTI version of the movement regressors lying around?  If it serves some purpose, can you add a comment explaining what its value is?